### PR TITLE
Add support for pre-population of Contents.swift

### PR DESCRIPTION
**Pull request by cameroncooke**
I've modified the script to support the pre-populatation of  the `Contents.swift` from input passed via stdIn.

This means that you can essentially start with a standard template e.g.

```bash
cat source.swift | playground
```

Running `playground` without piping input will result in the script behaving exactly as it does currently. The user will never be prompted for input.

My actual use-case for this was because I wanted to create an Automator script that passes in the selected text to the script via stdIn and creates a playground with the Contents.swift pre-populated with the selected text. 

I then use this service to allow me to select source-code examples in iBooks and paste them into a new Playground. This is because iBooks on the Mac prevents copy & paste but allows macOS services to process selected text.

screen shot 2017-04-04 at 13 47 42

Which is as simple as creating the automator service above and running the updated `playground` script.

I've also modified the script to add the time to the filename so that a new playground is created each time. Though just thinking now about how this might be used it might be better to add an option for this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Simply run `playground` in your terminal, and you’ll have a playground opened
 To install `playground`, simply run the following command in your terminal of choice:
 
 ```
-$ curl https://raw.githubusercontent.com/JohnSundell/playground/master/playground > /usr/local/bin/playground; chmod +x /usr/local/bin/playground
+$ curl https://raw.githubusercontent.com/cameroncooke/playground/master/playground > /usr/local/bin/playground; chmod +x /usr/local/bin/playground
```
Comment (JohnSundell) [inline README.md:15]: Hey! 😉

**Review by JohnSundell — COMMENTED** [review]
Hi @cameroncooke!

Thanks for contributing to `playground`! 👍

The change looks really good - it's definetly a feature that the script should support. However, I'm actually right about to rewrite in in Swift - so unsure whether any changes should be made to the shell-based implementation before that.

Also, I personally wouldn't want the time to always be appended to the playground's name, since I use this script a lot (like 10+ times per day), and love the feature that it either creates or opens an existing playground for the day.

I'm planning to add the ability to set the path/name of the created playground through an argument though, in the Swift implementation 🙂

**Comment by JohnSundell** [comment]
Hey @cameroncooke! I've now rewritten `playground` in Swift, and added support for pre-populating the playground (using the `-c` argument). Thanks again for contributing and hope you like the new feature 🎉 Going to close this now. Feel free to open new PRs for any new features you'd like to see in `playground`.

File: README.md (hunk 1 above)
Comment (cameroncooke) [reply README.md:15]: Yeah my bad, I changed that after making the pull-request.

**Comment by cameroncooke** [comment]
That's awesome, nice work.

**Pull request closed without merging**